hooks/post-rewrite.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+./hooks/git-format-staged --formatter 'mint run swiftformat --config .swiftformat stdin' 'Sources/*.swift | grep -v "Generated'
+./hooks/git-format-staged --formatter 'mint run swiftformat --config .swiftformat stdin' 'Tests/*.swift'
+./hooks/git-format-staged --formatter 'mint run swiftformat --config .swiftformat stdin' 'Sample/*.swift'
+./hooks/git-format-staged --formatter 'mint run swiftformat --config .swiftformat stdin' 'DemoApp/*.swift'
+./hooks/git-format-staged --formatter 'mint run swiftformat --config .swiftformat stdin' 'Integration/*.swift'
+./hooks/git-format-staged --formatter 'mint run swiftformat --config .swiftformat-snippets stdin' 'DocsSnippets/*.swift'
+
+# Regenerage Package.swift files if needed
+./hooks/regenerage-spm-package-if-needed.sh
```

Comment:
This is just waiting for someone to update just one of the hooks. See my comment above about using the same script for both,

Reply:
Yea, deleted the file and linked the pre-commit to the post-rewrite hook as well...